### ISSUE TITLE
Incorrect response when body is a single number

### ISSUE THREAD

**Issue by ssams** [jormaechea/open-api-mocker#56]
When the response is a single number, its accidentally sent as status code, creating an invalid response which has an unexpected status and no body.

### Details

I have an OpenAPI spec (3.0) with an endpoint (simple get without parameters) using the following response:

```json
       "responses": {
          "200": {
            "content": {
              "application/json": {
                "schema": {
                  "type": "integer",
                  "example": 210
                }
              }
            }
          }
        }
```

When accessing the endpoint, the response sent from the mock server uses the HTTP status 210 (instead of 200 as specified), and contains no body (instead of the expected number "210").

The following information is logged by the mock server:

```
[INFO ] > [GET] /demo
[WARN ] * Missing content-type header
[DEBUG] Searching first response
[INFO ] < [200] 210 (13 ms)
express deprecated res.send(status): Use res.sendStatus(status) instead ..\node_modules\open-api-mocker\lib\mocker\express\server.js:184:14
```

open-api-mock version: *1.11.0*

### Source of the problem / suggested fix

The issue, as indicated by the logged message from the express server, seems to be caused by the invokation: https://github.com/jormaechea/open-api-mocker/blob/1101e9f303c7048a765e13fc826f76db1b50d195/lib/mocker/express/server.js#L184

As per the express API spec, the body parameter can be "a Buffer object, a String, an object, Boolean, or an Array" (https://expressjs.com/en/4x/api.html#res.send). In this case however it's a `number`, which apparently is used as status code due to the older/deprecated API.

So it should be ensured that the passed body is one of the listed types. A simple check like `return res.send(typeof body === 'number' ? body.toString() : body);` is sufficient to fix the problem in the described case.

**Comment by NaincyKumariKnoldus** [comment]
Hey @ssams can I do this please?

**Comment by ssams** [comment]
> Hey @ssams can I do this please?

you mean setup a pull request for it? sure, I just didn't get to it yet, the fix should be pretty simply anyway unless I missed something 😉

**Comment by NaincyKumariKnoldus** [comment]
yes I would like to raise a PR for it.

**Comment by jormaechea** [comment]
@ssams, @NaincyKumariKnoldus, @malteburkert sorry for the delay. Been really busy lately.
PR looks good, but coding style checks (`eslint`) are failing.

@NaincyKumariKnoldus please fix the error and I'll be happy to merge it.

**Comment by NaincyKumariKnoldus** [comment]
Hey @jormaechea I have tried to fix it. Please review it now.

**Comment by jormaechea** [comment]
@NaincyKumariKnoldus you're suposed to fix the code issue, not change the eslint configuration.

Please rollback your [last change](https://github.com/jormaechea/open-api-mocker/pull/57/files/d875c3966d920b06d60ed2412b2dc1f13924b047..75db2e9c381f8f711ee1588f077e7ad138698a08#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R27) and fix the code indentation so it includes tabs only.

**Comment by jormaechea** [comment]
@ssams, this has been merged in #57 and released as `v1.11.1`, please test it and let me know.

/cc @malteburkert @NaincyKumariKnoldus
